index.html

```diff
@@ -243,26 +243,43 @@ <h3>Sampling and Reporting Rate</h3>
 
 <section> <h2>Platform primitives</h2>
   <p>
-    The [=platform collector=] refers to a platform interface, with which the [=user agent=] interacts to
-    obtain the telemetry readings required by this specification.
+    A <dfn>pressure source</dfn> is an abstract, [=implementation-defined=]
+    interface to hardware counters or an underlying telemetry framework that
+    provides information about a <dfn>source types</dfn>
+    defined by {{PressureSource}}. A [=pressure source=] can make use of data
+    fusion with data from additional sources if that provides better results.
   </p>
   <p>
-    A [=platform collector=] can be defined by the underlying platform (e.g. in a native telemetry
-    framework) or by the [=user agent=], if it has a direct access to hardware counters.
+    A <dfn>platform collector</dfn> is an abstract interface responsible for
+    obtaining telemetry samples from a [=pressure source=], translating them
```

Comment:
Some places we say data, then telemetry or information. Not sure if we should make these more consistent or not

Reply:
Not sure translating is the best word. Transforming?

Reply:
I've replaced all uses of "translate" with "transform" in commit 5c966ff.

Reply:
> Some places we say data, then telemetry or information. Not sure if we should make these more consistent or not

I'm attempting to settle on "telemetry data" and, in some cases, just "data" in commit a2cf3a4.